_data/events.yml

```diff
@@ -1347,6 +1347,23 @@
   twitter: "https://twitter.com/itenium_be/status/1574320233984114688"
   githubUrl: "https://github.com/itenium-be/UnitTesting"
 
+
+- name: "CO2duino"
+  subTitle: "CO2 Meter"
+  date: 2022-06-29
+  track: "TechEvent"
+  summary: >
+       Monitored and managed your indoor air quality with 
+       precision using the CO2 Meter—your key to a healthier, 
+       more comfortable environment.
```

Comment:
Can you check what was posted on social media?
Here the past tense vs current tense doesn't make much sense